pkg/api/const.go

```diff
@@ -248,7 +248,7 @@ const (
 	APIVersionCompute             = "2019-07-01"
 	APIVersionDeployments         = "2018-06-01"
 	APIVersionKeyVault            = "2018-02-14"
-	APIVersionManagedIdentity     = "2015-08-31-preview"
+	APIVersionManagedIdentity     = "2018-11-30"
```

Comment:
@xuto2 @yangl900 Does AKS rely upon aks-engine for its Managed Identity implementation?

FYI above

Reply:
I can revert that api version change if it is going to cause issues. The only reason I changed it was because I saw the hardcoded version 2018-11-30 elsewhere in the codebase and wanted to clean up some potential debt.

Reply:
@yangl900 @xuto2 FYI, this PR will change the API version for managed identity to 2018-11-30